sdk/core/azure-core/azure/core/polling/_poller.py

```diff
@@ -33,12 +33,12 @@
 from azure.core.tracing.common import with_current_context
 
 
-PollingReturnType = TypeVar("PollingReturnType")
+PollingReturnType_co = TypeVar("PollingReturnType_co", covariant=True)
```

Comment:
Do we need the reverse Hungarian notation for the type parameter?

Reply:
@johanste this is recommendation from typing doc and @kristapratico